src/w3d/renderer/linegroup.h

```diff
@@ -0,0 +1,114 @@
+/**
+ * @file
+ *
+ * @author tomsons26
+ *
+ * @brief
+ *
+ * @copyright Thyme is free software: you can redistribute it and/or
+ *            modify it under the terms of the GNU General Public License
+ *            as published by the Free Software Foundation, either version
+ *            2 of the License, or (at your option) any later version.
+ *            A full copy of the GNU General Public License can be found in
+ *            LICENSE
+ */
+#pragma once
+
+#include "always.h"
+#include "shader.h"
+#include "sharebuf.h"
+#include "vector3.h"
+#include "vector4.h"
+
+class RenderInfoClass;
+class TextureClass;
+
+class LineGroupClass
```

Comment:
Are there no hooks required for this?

Reply:
uh so this is only called by w3d particle classes, meaning we can only test it when w3d particles are in and @jonwil can do a battery of tests as he's the most familiar with this system.
So im not sure should i bother at this time as none of this would be called anyway..